api/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@microsoft/vscode-azureresources-api",
+    "version": "2.0.0",
+    "description": "Type declarations and client library for the Azure Resources extension API",
+    "main": "out/index.js",
+    "types": "vscode-azureresources-api.d.ts",
+    "scripts": {
+        "build": "tsc -p ./",
+        "compile": "tsc -p ./ -watch",
+        "prepack": "npm run build",
+        "api-extractor": "npm run build && api-extractor run"
+    },
+    "author": "Microsoft",
+    "license": "MIT",
+    "devDependencies": {
+        "@azure/ms-rest-azure-env": "^2.0.0",
+        "@microsoft/api-extractor": "^7.33.8",
+        "@microsoft/vscode-azext-utils": "^0.4.0",
```

Comment:
Ideally this wouldn't depend on `utils` at all. However the activity types are unable to be moved here because they rely on `AzExtTreeItem`, and I really don't want that here.

Reply:
But it's not a huge deal since it's just a dev dependency.